industrial_taxonomy/getters/official.py

```diff
@@ -0,0 +1,109 @@
+# Data getters for official data
+
+from functools import lru_cache
+from typing import Dict, Optional
+
+from metaflow import Flow, Run
+from metaflow.exception import MetaflowNotFound
+
+import pandas as pd
+
+
+@lru_cache()
+def get_run(flow: str):
+    """Last successful run executed with `--production`.
+
+    Arguments:
+        flow: the official data flow we want to get
+    """
+    runs = Flow(flow).runs("project_branch:prod")
+    try:
+        return next(filter(lambda run: run.successful, runs))
+    except StopIteration as exc:
+        raise MetaflowNotFound("Matching run not found") from exc
+
+
+def gva_lad():
+    """get the GVA in a local authority
+
+    Columns:
+        Name: itl1_region, dtype: str, NUTS1 region (e.g. Scotland, South East etc)
+        Name: la_code, dtype: str, local authority code
+        Name: la_name, dtype: str, local authority name
+        Name: year dtype: int, year (ranges between 1998 and 2019
+        Name: gva dtype: float, £M Gross value added
+    """
+
+    return pd.DataFrame(get_run("LocalGdpData").data.gva).melt(
+        id_vars=["itl1_region", "la_code", "la_name"], var_name="year", value_name="gva"
+    )
+
+
+def population_lad():
+    """get the population in a local authority
+
+    Columns:
+        Name: itl1_region, dtype: str, NUTS1 region (e.g. Scotland, South East etc)
+        Name: la_code, dtype: str, local authority code
+        Name: la_name, dtype: str, local authority name
+        Name: year dtype: int, year (ranges between 1998 and 2019
+        Name: pop dtype: float, population
+    """
+
+    return pd.DataFrame(get_run("LocalGdpData").data.pop).melt(
+        id_vars=["itl1_region", "la_code", "la_name"], var_name="year", value_name="pop"
+    )
+
+
+def gva_pc_lad():
+    """Get the GVA per capita in a local authority
+
+    Columns:
+        Name: itl1_region, dtype: str, NUTS1 region (e.g. Scotland, South East etc)
+        Name: la_code, dtype: str, local authority code
+        Name: la_name, dtype: str, local authority name
+        Name: year dtype: int, year (ranges between 1998 and 2019
+        Name: gva dtype: float, GDP per capita
+    """
+
+    gva = gva_lad()
+    pop = population_lad()
+
+    return (
+        gva.merge(pop, on=["itl1_region", "la_code", "la_name", "year"])
+        .assign(gdp_pc=lambda df: (1e6 * df["gva"] / df["pop"]).round(2))
+        .drop(axis=1, labels=["gva", "pop"])
+    )
+
+
+def nomis():
```

Comment:
Q: from a user perspective - would it be better to have a getter for each indicator? i.e. will they be used all together or not?